apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -972,6 +972,46 @@ const Ticket = new GQLListSchema('Ticket', {
         delete: false,
         auth: true,
     },
+    kmigratorOptions: {
+        indexes: [
+            {
+                type: 'BTreeIndex',
+                fields: ['unitName'],
+                name: 'ticket_unitName',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['isPayable'],
+                name: 'ticket_isPayable',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['isEmergency'],
+                name: 'ticket_isEmergency',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['isWarranty'],
+                name: 'ticket_isWarranty',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['statusReopenedCounter'],
+                name: 'ticket_statusReopenedCounter',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['assignee', 'executor', 'organization'],
+                name: 'ticket_assignee_exec_org',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['organization', 'status'],
+                name: 'ticket_organization_status',
+                condition: 'Q(deletedAt__isnull=True)',
```

Comment:
Why is the condition `deletedAt` specified here, but not for other indexes?

Reply:
this query comparing to other simple indexes had deletedAt: null